gix-blame/src/file/function.rs

```diff
@@ -61,6 +61,7 @@ pub fn file<E>(
     traverse: impl IntoIterator<Item = Result<gix_traverse::commit::Info, E>>,
     resource_cache: &mut gix_diff::blob::Platform,
     file_path: &BStr,
+    range: Option<Range<u32>>,
```

Comment:
`range` would need to be documented. Given how 'special' it is no details should be spared :).

Reply:
So many helpful comments! :-) I addressed all of them.